website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -11,72 +11,97 @@ This page provides steps to install Appsmith on a Kubernetes cluster using the [
 
 <VideoEmbed host="youtube" videoId="wZzYL1uZwds" title="How To Deploy Appsmith On Kubernetes" caption="How To Deploy Appsmith On Kubernetes" />
 
-### Prerequisites
-1. Ensure that you have at least 15 GB of free storage space and 4 GB of RAM.
-2. [Helm Package Manager](https://helm.sh/docs/intro/install/) to install and manage applications on Kubernetes clusters.
-3. [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) to manage resources on Kubernetes clusters.
-4. Follow the steps below to install Appsmith on your Kubernetes cluster:
+## Prerequisites
+1. At least 15 GB of free storage space.
+2. 4 GB of RAM.
 
-<Tabs groupId="configurations" queryString="current-configuration">
- <TabItem label="AWS EKS" value="AWSEKS">
+## Before you begin
+1. Installed Helm package manager on your local machine. See the official Helm documentation for your [operation system](https://helm.sh/docs/intro/install/#through-package-managers).
+2. A running Kubernetes cluster. If not present, follow these steps to set up the cluster:
 
-  a. Set up the Kubernetes cluster by [creating a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).
+    a. Verify if you have access to AWS CLI with:
+      
+    ```bash
+      aws sts get-caller-identity 
+    ```
+    
+    The above command provides information related to your account and ARN, indicating that you can connect and access your Amazon account using a terminal.
 
-  b. Ensure that you have a default storage class running. If not, [create default storage class](https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html).
+    b. Create KubeConfig with: 
 
-</TabItem>
-<TabItem label="Google Cloud Kubernetes" value="GoogleCloudKubernetes">
+    ```bash
+    aws eks update-kubeconfig --region ap-south-1 --name CLUSTER_NAME  --profile PROFILE_NAME
+    ```
+    In the above command, add the profile name that has access to the EKS cluster to the `--profile` parameter.
 
-  a. [Configure cluster access for kubectl](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl) on Google Cloud Kubernetes.
+    c. Test your Kubernetes configuration with:
 
-  b. Ensure that you have a default storage class running. If not, [set up a default storage class on Google Kubernetes Engine](https://cloud.google.com/anthos/clusters/docs/on-prem/1.3/how-to/default-storage-class)[GKE].
+    ```bash
+    kubectl cluster-info
+    ```
+    The above command provides a summary of the current cluster configuration, including the Kubernetes master and other cluster information. 
 
-</TabItem>
-<TabItem label="Microk8s" value="Microk8s">
+    d. A default storage class running. If not created yet, create the storage class with:
 
-  a. Set up Kubernetes by [working with kubectl](https://microk8s.io/docs/working-with-kubectl).
+    ```bash
+    kubectl apply -f - <<EOF
```

Comment:
@pratapaprasanna - Please update the storage file command based on our discussion.

Reply:
**for Kubernetes Version <= 1.20
we can use this 
for Kubernetes Version >= 1.23**

```
helm repo add aws-ebs-csi-driver https://kubernetes-sigs.github.io/aws-ebs-csi-driver
helm repo update
helm upgrade --install aws-ebs-csi-driver \
    --namespace kube-system \
    aws-ebs-csi-driver/aws-ebs-csi-driver
```
**Verify the installation using** 
```
kubectl get pods -n kube-system -l app.kubernetes.io/name=aws-ebs-csi-driver
```